modules/userId/index.js

```diff
@@ -27,7 +27,7 @@
  *  If IdResponse#callback is defined, then it'll called at the end of auction.
  *  It's permissible to return neither, one, or both fields.
  * @name Submodule#extendId
- * @param {SubmoduleParams} configParams
+ * @param {SubmoduleParams} config
```

Comment:
Here `SubmoduleConfig` instead of `SubmoduleParams`?

Reply:
yep, i forgot i changed extendId too, good catch